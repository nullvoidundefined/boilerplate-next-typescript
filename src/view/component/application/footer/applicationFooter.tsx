import styled from "styled-components";
import { APPLICATION_COLOR, APPLICATION_DIMENSION } from "../../../../constant";

const StyledApplicationFooter = styled.footer`
  background-color: ${APPLICATION_COLOR.FOOTER_BACKGROUND};
  height: ${APPLICATION_DIMENSION.FOOTER_HEIGHT}px;
`;

const ApplicationFooter = () => {
  return <StyledApplicationFooter>Footer</StyledApplicationFooter>;
};

export { ApplicationFooter };
