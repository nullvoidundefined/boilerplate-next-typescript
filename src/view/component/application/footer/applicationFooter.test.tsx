import { ApplicationFooter } from "./applicationFooter";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const footer = renderer.create(<ApplicationFooter />).toJSON();
  expect(footer).toMatchSnapshot();
});
