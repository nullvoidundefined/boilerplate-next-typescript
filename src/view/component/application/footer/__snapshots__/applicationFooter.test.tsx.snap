// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="sc-17d65132-0 gWzca-D"
>
  Footer
</footer>
`;
